RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -1510,6 +1510,19 @@ boolean L13_towerNSTower()
 			}
 		}
 		
+		//Wall Of Bones combat uses Unleash The Greash, Garbage Nova, or Saucegeyser
+		if(!auto_have_skill($skill[Garbage Nova]) && have_effect($effect[Takin\' It Greasy]) == 0)
+		{
+			float saucegeyserDamage = MLDamageToMonsterMultiplier()*ceil((numeric_modifier("Spell Damage Percent")/100.0)*(60 + numeric_modifier("Spell Damage") + numeric_modifier("Hot Spell Damage") + 0.4*my_buffedstat($stat[mysticality])));
```

Comment:
not familiar with wall of bones killing. I don't see it has an element alignment. Would it make sense to instead of always adding hot spell damage, add max of cold or hot as geyser will use whichever element will do more damage?

Reply:
You're right, changed